src/litdata/streaming/downloader.py

```diff
@@ -144,7 +181,13 @@ def download_file(self, remote_filepath: str, local_filepath: str) -> None:
         super().download_file(remote_filepath, local_filepath)
 
 
-_DOWNLOADERS = {"s3://": S3Downloader, "gs://": GCPDownloader, "local:": LocalDownloaderWithCache, "": LocalDownloader}
+_DOWNLOADERS = {
+    "s3://": S3Downloader,
+    "gs://": GCPDownloader,
+    "azure://": AzureDownloader,
```

Comment:
Hi @tchaton, @jaehwana2z,

I noticed the use of the `azure://` prefix, which seems totally perfect for this context. 
I'm curious if there are any conventions or considerations for using this prefix. Additionally, would it be worth exploring other possible prefixes, such as `azs://`, `bs://`, or `abs://`?

Thank you!

Reply:
We follow the prefix from the cloud providers. `azure` seems the official one.

Reply:
But we should add support for all most common ones.

Reply:
Thank you for the insights on the prefix, @tchaton. 

I will follow up about the cloud providers in another issue or on Discord.